api/pkg/app/app.go

```diff
@@ -83,9 +83,12 @@ type EnvMode string
 
 // Types of EnvMode
 const (
-	Production  EnvMode = "production"
-	Development EnvMode = "development"
-	Test        EnvMode = "test"
+	Production             EnvMode = "production"
+	Development            EnvMode = "development"
+	Test                   EnvMode = "test"
+	ConfigFileDir          string  = "/tmp/config/"
```

Comment:
we'll probably have to mark this constant in the docs so that if somebody wants to deploy their hub then this value shouldn't change in deployment.yaml